doc/nrf/app_dev/device_guides/nrf54l.rst

```diff
@@ -1,7 +1,8 @@
 .. _ug_nrf54l:
+.. _ug_nrf54L15_gs:
 
-Working with nRF54L Series
-##########################
+Developing with nRF54L Series
```

Comment:
Apologies, @annwoj , I forgot to mention. This change also requires moving files from `device_guides/working_with_nrf/nrf54l` one level up, to `device_guides/nrf54l`. 

This change requires edits to the redirects.py and ncs.tags.yaml files.